packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -121,6 +121,8 @@ void testWidgets(
       return binding.runTest(
         () async {
           debugResetSemanticsIdCounter();
+          tester.testTextInput.register();
```

Comment:
Should there maybe be a convenience method and tester (or testTextInput) to reset everything that does both of this?

Reply:
I wonder if this is the right place to reset these things, it feels odd here.

Reply:
I tried doing it before the `binding.runTest` and that didn't work.  FWIW, this is where we reset the semantics ID counter as well so...

I'll move these to a convenience method.